Integration/Orleans.InProcess/OrleansFixture.cs

```diff
@@ -118,7 +118,7 @@ protected override void Dispose(bool disposing)
             _backupPerformed = true;
         }
         GlobalFixture.RemoveAllDatabases().GetAwaiter().GetResult();
-        base.Dispose(disposing);
+        base.Dispose(false);
```

Comment:
This sped up the integration tests a lot! Though there is probably something in Chronicle which causes it to be so slow on shutdown that we have to figure out some time